yarn.lock

```diff
@@ -9820,6 +9820,51 @@ liferay-npm-bundler@2.19.4:
     xml-js "^1.6.8"
     yargs "^14.0.0"
 
+liferay-theme-tasks@10.1.0:
```

Comment:
I'm not confident about this... @wincent, did you send something to fix or not-fix but something this?

Reply:
Yes. It looked correct in a first small analysis, but I'm not quite sure of why we get this. :thinking:

Reply:
I guess this is fixed by #371 :thinking:

Reply:
Yep, that's right.

> I'm not quite sure of why we get this

799f34240f5d6139e93582af6a530040ac77cb6a shows what Yarn is complaining about. If we release a new version of a workspace project (ie. in this case, `liferay-theme-tasks@10.2.0`) but continue to have an internal dependency on the old version of the project (ie. `liferay-theme-tasks@10.1.0`), Yarn is going to drag the old version into the lockfile. The fact that Yarn complains (and that CI surfaces the complaint) is a good thing, because it shows us that we forgot to update an internal reference. And the stale reference is a problem not just because of lockfile noise but because it means we have two copies of our own dependency floating around in the worktree; in this case:

- `10.2.0` at `projects/js-themes-toolkit/packages/liferay-theme-tasks`; and:
- `10.1.0` at `projects/npm-tools/packages/npm-scripts/node_modules/liferay-theme-tasks`

We don't want that kind of thing in general because it makes us look fat, and in this specific instance because it means the improvement added in 10.2.0 (#365) might get overlooked and not actually make it into liferay-portal in a timely fashion.

> I guess this is fixed by

Yep, 5145c8d087f8a1ae129f090cb06d3b870d08d6f1 just updates the internal reference and the redundancy goes away.

Reply:
Strange. I think I also ran `yarn why` (it's the first I instinctively do when this things happen) but didn't report any duplicates. However it was some time ago, so maybe I didn't do it... :shrug: